.github/workflows/pkg.pr.new.yml

```diff
@@ -37,4 +37,6 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: output.json
-          path: output.json
+          path: ./output
```

Comment:
I just saw this. I'm pretty sure this needs to be the path to the file you want to save, I don't think this will actually save the JSON file

Reply:
Name is just a reference, it can be anything, more like an id. Path is the thing that determines what gets saved.

Reply:
Yeah: https://github.com/sveltejs/svelte/actions/runs/11685581647/job/32539351892#step:9:1